.papr.yml

```diff
@@ -55,6 +55,8 @@ env:
 # present, tests will run after a successful build. Full
 # UTF-8 values are supported.
 tests:
+    - yum install -y --skip-broken pylint
+    - yum downgrade -y pylint
```

Comment:
Yes, this is just a 'see if it works' thing.  Skookum comments (with spell-checking) can come later 😁